src/Components/Web.JS/src/DomWrapper.ts

```diff
@@ -5,7 +5,7 @@ import '@microsoft/dotnet-js-interop';
 
 export const domFunctions = {
   focus,
-  focusBySelector,
+  focusOnNavigate
```

Comment:
I don't think this change is required any more - the old name was slightly more descriptive since in theory this could be called at any time, not just on navigation. However I don't really mind if you prefer the new name!